packages/webview_flutter/example/integration_test/webview_flutter_test.dart

```diff
@@ -532,6 +535,7 @@ void main() {
       expect(fullScreen, _webviewBool(false));
     });
 
+    // allowsInlineMediaPlayback is a noop on Android, so it is skipped.
```

Comment:
https://github.com/flutter/plugins/blob/master/packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java#L397

This test fails when running on Android, but it doesn't seem like there is anything we can do since a flag like this isn't supported on Android. At least according to the comment.